<?xml version="1.0"?>

<launch>
    <node name="case_three" pkg="ass3" type="case_three.py"   />
    <remap from="cmd_vel" to="remap_cmd_vel" />
    <node name="teleop_twist_keyboard" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py"  output="screen" />
</launch>